Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -670,36 +670,44 @@ In DSR with Geneve, Cilium encapsulates packets to the Loadbalancer with the Gen
 header that includes the service IP/port in the Geneve option and redirects them
 to the backends.
 
-The Helm example configuration in a kube-proxy-free environment with DSR and
-Geneve dispatch enabled would look as follows:
+DSR with Geneve is compatible with the Geneve encapsulation mode (:ref:`arch_overlay`).
+It works with either the direct routing mode or the Geneve tunneling mode.
```

Comment:
Reading this a couple of times I don't think it distinguished the settings clearly enough to communicate the ideas clearly to readers. I suggest this alternate wording:

```suggestion
You can use DSR with Geneve for encapsulating LoadBalancer traffic regardless                                                                                                                                                                                  
of whether you have configured the routing mode to route traffic using Geneve                                                                                                                                                                                  
:ref:`encapsulation` or :ref:`native_routing`.
```